app/src/main/res/values-ar/strings.xml

```diff
@@ -342,10 +342,10 @@
     <string name="description_post_cw">تحذير عن المحتوى: %s    </string>
     <string name="description_post_media_no_description_placeholder">مِن دون وصف    </string>
     <string name="description_post_reblogged">أعاد تدوينه    </string>
-    <string name="description_visiblity_public">للعامة    </string>
-    <string name="description_visiblity_unlisted">غير مُدرَج    </string>
-    <string name="description_visiblity_private">المتابِعون    </string>
-    <string name="description_visiblity_direct">مباشر    </string>
+    <string name="description_visibility_public">للعامة    </string>
+    <string name="description_visibility_unlisted">غير مُدرَج    </string>
+    <string name="description_visibility_private">المتابِعون    </string>
+    <string name="description_visibility_direct">مباشر    </string>
```

Comment:
Some projects ask contributors to adjust localizations, others ask them not to. It's about 50/50. I've included the changes here, but it's trivial to drop them.